app/assets/stylesheets/govuk_publishing_components/components/_back-link.scss

```diff
@@ -1 +1,40 @@
 @import "govuk/components/back-link/back-link";
+
+.govuk-back-link {
+  &[href] {
+    text-decoration: underline;
+    border-bottom: 1px solid transparent;
+
+    &:focus {
+      text-decoration: none;
+    }
+  }
+
+  &:before {
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    bottom: 1px;
+    left: 3.31px;
+    width: 7px;
+    height: 7px;
+    -webkit-transform: rotate(225deg);
+    -ms-transform: rotate(2255deg);
+    transform: rotate(225deg);
+    border: solid;
+    border-width: 1px 1px 0 0;
+    clip-path: inherit;
+  }
+}
+
+@include govuk-media-query($until: tablet) {
```

Comment:
This may be a question for the team, but I wonder if we want the larger touch target to apply all the time including on desktop? I'm not sure what the recommendations are, but it seems feasible that someone could be viewing the site in desktop view but on a touchscreen device.

Reply:
Asked PM and response was "That makes sense to me - would that have any adverse effect on the look of anything around the Back link on desktop? If not, happy to go with that change for desktop too. " 

As it wouldn't affect desktop have removed media query so it was just on mobile